UPGRADE-4.x.md

```diff
@@ -1,13 +1,23 @@
 UPGRADE 4.x
 ===========
 
-UPGRADE FROM 3.x to 3.x
+UPGRADE FROM 4.x to 4.x
 =======================
 
 ### Support for NelmioApiDocBundle > 3.6 is added
 
 Controllers for NelmioApiDocBundle v2 were moved under `Sonata\UserBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
 
+### Fix REST API routing paths
+
+In `SonataUserBundle` v4.6.0 some extra REST API routing paths were added. You MUST avoid using them to keep API doc cleaner. To do so, make sure to use the right routing file:
```

Comment:
I'm sorry for not catching this earlier. In my opinion, this is not just about the API document, but about the whole routing, so I propose to use a message like this:
```suggestion
In version 4.6.0 some extra REST API routes were added by mistake, creating new duplicated paths pointing to existing actions (by instance `/groups/{id}.{_format}` was duplicated as `/group/{id}.{_format}`, `/users/{id}.{_format}` was duplicated as `/user/{id}.{_format}`, etc).
You MUST avoid importing these routes in order to keep your API routing clean and consistent with previous versions. To do so, make sure to import some of these routing files, depending on your needs:
```